agent/runner/jobs/pbm_helpers.go

```diff
@@ -30,11 +30,13 @@ import (
 
 const (
 	// How many times check if backup/restore operation was started
-	maxBackupChecks     = 10
-	maxRestoreChecks    = 10
+	maxBackupChecks  = 10
+	maxRestoreChecks = 10
+
 	cmdTimeout          = time.Minute
 	resyncTimeout       = 5 * time.Minute
 	statusCheckInterval = 3 * time.Second
+	maxLogsChunkSize    = 50
```

Comment:
That's not specific for mongo, later we will use it for other DB types. I think it should stay in `common.go`

Reply:
I think we can move it whenever we start to use it somewhere else.